scss/atoms/typography/_typography.scss

```diff
@@ -5,3 +5,48 @@
 @import
 	"variables",
 	"functions";
+
+meter {
+  /* Reset the default appearence */
+  -webkit-appearance: none;
+  width: 550px;
+  height: 25px;
```

Comment:
This indentation is wrong (we don't have a SCSS style guide yet but it's kind of already decided that we'll use tabs. It's inconsistent with the rest of the code anyways.)